src/Symfony/Component/HttpClient/DataCollector/HttpClientDataCollector.php

```diff
@@ -43,8 +43,8 @@ public function collect(Request $request, Response $response, \Throwable $except
 
     public function lateCollect()
     {
-        $this->data['request_count'] = 0;
-        $this->data['error_count'] = 0;
+        $this->data['request_count'] = $this->data['request_count'] ?? 0;
```

Comment:
Maybe `$this->data['request_count'] ??= 0`

Reply:
Symfony 5.4 must be compatible with PHP 7.2.